app/javascript/lib/validation/validate-file-content.js

```diff
@@ -212,12 +224,11 @@ function sniffDelimiter(lines, mimeType) {
  * Cap lines are like meta-information lines in other file formats
  * (e.g. VCF), but do not begin with pound signs (#).
  */
-async function validateCapFormat(table, fileType) {
+function validateCapFormat([headers, annotTypes]) {
   let issues = []
-
-  // Trim spaces and quotes
-  const headers = table[0].map(header => clean(header))
-  const annotTypes = table[1].map(type => clean(type))
+  if (!headers || !annotTypes) {
+    return [['error', 'cap:format:no-header', 'File does not have at least 2 non-empty header rows']]
```

Comment:
Blocker: The use of `cap:format` instead of `format:cap` is a bug that had slipped by in earlier work.  

And is "at least" correct?  To my memory, metadata and cluster files must have two _and only two_ such rows.

```suggestion
    return [['error', 'format:cap:no-header', 'File does not have 2 non-empty header rows']]
```